deps/bzip2.BUILD.bazel

```diff
@@ -24,6 +24,10 @@ license(
     visibility = ["//visibility:public"],
 )
 
+exports_files([
```

Comment:
We should never have to do this.  Leave it out if it is is not breaking tests.
It's a Googleism that has cargo-culted.